cassovary-core/src/main/scala/com/twitter/cassovary/graph/DirectedPathCollection.scala

```diff
@@ -54,11 +54,10 @@ class DirectedPathCollection {
 
   /**
    * @return an map of top DirectedPaths with occurrence
-   *         ending at {@code node}, sorted decreasing
+   *         ending at `node`, sorted decreasing
```

Comment:
comment is still not clear. How about
@return a map of {path -> count} for paths ending at node, sorted in decreasing order of count. Paths with the same count are sorted shortest path first.